client/app/scripts/liveblog-edit/module.js

```diff
@@ -194,8 +194,8 @@ define([
         });
     }
 
-    BlogSettingsController.$inject = ['$scope', 'blog', 'api', 'blogService', '$location', 'notify'];
-    function BlogSettingsController($scope, blog, api, blogService, $location, notify) {
+    BlogSettingsController.$inject = ['$scope', 'blog', 'api', 'blogService', '$location', 'notify', 'gettext'];
+    function BlogSettingsController($scope, blog, api, blogService, $location, notify, gettext) {
```

Comment:
it was used a bit later with notify.info(gettext('saving blog settings')); and travis was showing an error from time to time.... something like gettext module is not imported